charts/gp-reporting-kafka-health/v0.0.1/questions.yml

```diff
@@ -0,0 +1,162 @@
+questions:
+  # General environment settings
+  - variable: environment.parentName
+    type: enum
+    required: true
+    group: "Environment"
+    label: Environment Parent
+    description: The name of the parent environment we will deploy into and funcation as.
+    options:
+      - sandbox
+      - canary
+      - production
+    default: sandbox
+
+  - variable: environment.modifierValue
+    type: enum
+    required: false
+    group: "Environment"
+    label: Environment Modifier
+    description: The modifier name for the environment we will deploy into and funcation as.
+    options:
+      - prime
+      - qa
+      - testing
+      - 01
+      - 02
+      - 03
+      - 04
+      - 05
+      - 06
+      - 07
+      - 08
+      - 09
+      - 10
+    default: "01"
+
+  ####
+  #### Reporting Kafka Health Service Settings
+  ####
+  - variable: build.commitSHA
+    type: string
+    required: true
+    group: "Application"
+    label: Application Build SHA
+    description: The commit SHA of the GitPrime Application build you want to deploy.
+    default: latest
+
+  - variable: volumes.enableLocalMount
```

Comment:
For this app you can probably skip the volumes stuff one if there's no local storage.

Reply:
Remove all volumes configuration